omod/src/main/webapp/resources/htmlforms/096a-HealthUnitTBRegister-SummaryPage.xml

```diff
@@ -733,7 +948,7 @@
 				<td>
 					<label>X-ray results</label>
 					<obs 	conceptId="12" 

```

Comment:
@moshonk Leave this uuid as the version that I had changed it to since dc9816bd-30ab-102d-86b0-7a5022ba4115 is not available in the TB concepts for 1115 and was changed to CIEL otherwise it breaks the forms